openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -67,12 +67,16 @@
 - control_key: IA-2 (4)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-303
+        OCP offloads (via: https://access.redhat.com/documentation/en-us/
+        openshift_container_platform/4.3/html/authentication/
+        understanding-identity-provider)
+        responsibility for authentication, session management, and OAuth
+        controls to the hosting environment’s LDAP and leverages CAC/PKI,
+        FIPS 140-2 compliant ciphers, SSO and does not provide for group
+        accounts.
```

Comment:
```suggestion
        OpenShift, as a web application, is managed remotely through
        a web console and/or API endpoints. The implementation
        of multifactor authentication for local access to non-privileged
        accounts is not applicable to OpenShift.
```

Reply:
@kevindoingbizas Would you be OK with this suggested language? Aligned with comments in https://github.com/ComplianceAsCode/redhat/pull/1185